Mathlib/Topology/Algebra/Group/Basic.lean

```diff
@@ -575,10 +575,16 @@ alias tendsto_inv_nhdsWithin_Iic_inv := tendsto_inv_nhdsLE_inv
 
 end OrderedCommGroup
 
-@[to_additive]
```

Comment:
Can you write `@[to_additive instIsTopologicalAddGroupProd]` instead? In case that doesn't work, what happens?

Reply:
@grunweg, this isn't necessary with my suggestion above